### ISSUE TITLE
Failing to Build .NET 7.0 project.

### ISSUE THREAD

**Issue by joshberc** [railwayapp/nixpacks#813]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

When deploying a .NET 7.0 app to railway, receiving the following error:
`/nix/store/832ihvqk3vxgqqs5hvcyvg6bxqybky14-dotnet-sdk-6.0.403/sdk/6.0.403/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.TargetFrameworkInference.targets(144,5): error NETSDK1045: The current .NET SDK does not support targeting .NET 7.0.  Either target .NET 6.0 or lower, or use a version of the .NET SDK that supports .NET 7.0. [/app/Project-90-API.csproj]`

Can be resolved by adding the .NET 7 SDK to the nix store. 

### To reproduce

1. Create a template .NET Core 7.0 Web API. 
2. Upload to github.
3. Deploy to Railway and wait for build.

### Expected behavior

Should build without error. Can be resolved by adding the .NET 7 SDK to the nix store. 

### Environment

Nixpack v 1.4.0 via Railway
.NET core 7.0

**Comment by Milo123459** [comment]
can you send us a link to your project

**Comment by joshberc** [comment]
https://github.com/joshberc/Project-90-API It's a template .NET 7.0 Web API.

**Comment by Milo123459** [comment]
can you try set version in global.json?

**Comment by joshberc** [comment]
Updated. Nix output this error upon build: "Install the [7.0.200] .NET SDK or update [/app/global.json] to match an installed SDK."

**Comment by joshberc** [comment]
The deploy process is parsing both the global.json and project settings correctly. This should be resolved be adding the 7.0.200 SDK to /nix/store. @Milo123459 is it possible to add?

**Comment by MikePreston17** [comment]
Hi there!

I don't know if this might help, but I got 7.0 working on my Ubuntu 22.04 machine when I had issues installing - and switching to - dotnet 7.0 from 6.0.

Here's a shell script I use to get the correct Ubuntu SDK and Runtime.  Perhaps it will help y'all solve this:

```bash

# This contact's MS's version of dotnet for ubuntu
# Explanation is here: https://learn.microsoft.com/en-us/dotnet/core/install/linux-ubuntu#2204-microsoft-package-feed

wget https://packages.microsoft.com/config/ubuntu/22.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
sudo dpkg -i packages-microsoft-prod.deb
rm packages-microsoft-prod.deb

sudo apt-get update
sudo apt-get install -y dotnet-sdk-7.0
sudo apt-get install -y aspnetcore-runtime-7.0

## these are to be removed after installation
sudo apt autoremove aspnetcore-targeting-pack-6.0 dotnet-apphost-pack-6.0 dotnet-targeting-pack-6.0 gir1.2-snapd-1 libllvm13 libllvm13:i386
https://github.com/harriris/external-ballistics-js.git
echo "here'https://github.com/harriris/external-ballistics-js.gits the runtimes & sdks (if any):"

dotnet --list-runtimes
dotnet --list-sdks
```
